content/docs/faq.md

```diff
@@ -85,9 +87,9 @@ Anyway, back to the point. You can contribute in many ways, and some of it is re
 
   Are you familiar with Scratch Addons? If so, you can write the documentation for it. The documentation can include how to use it, or how it works. Please contact us on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions) for further information.
 
-- **Send a feedback**  
+- **Send feedback**  
   
-  You can send a feedback on our form, located at [the feedback page](https://scratchaddons.com/feedback). Your feedback may give us a different perspective in the extension development and help us know things needed attention and fix bugs.
+  You can send feedback on our form, located at [the feedback page](https://scratchaddons.com/feedback). Your feedback may give us a different perspective in the extension development and help us know things needed attention and fix bugs.
```

Comment:
I don't know, but maybe we should just put this  as "a feedback." The previous point is "Suggest an idea," so it is quite inconsistent if you do this.

Reply:
@Hans5958 I'm sending you a feedback on your review. My feedback is that Grammarly is sending me a feedback letting me know that the article before "feedback" is incorrect. If you would like to send me a feedback on this response, send it now.
![image](https://user-images.githubusercontent.com/72760579/124410550-bd4fa880-dd18-11eb-9c87-5e102b4dbb45.png)

Reply:
Just found out that "feedback" is an uncountable noun, so it doesn't use any article (a/an/the). That's really weird.